dev/build.image/profiles/webProfile9/features.xml

```diff
@@ -0,0 +1,11 @@
+<feature>webProfile-9.0</feature>
```

Comment:
Since this is a new packaging why do we need both webProfile 9.0 and 9.1?

Reply:
If you just do 9.1 then you should be able to remove `packageServerConflict = true` from packageOpenLibertyWebProfile9 in build.image/build.gradle